### PR TITLE
migrate to-do app to this repo

### PR DESCRIPTION
**Pull request by Mukaba**
### Does my coding partner follow JS best practices?

### CODE REVIEW

**Review by Javed-Shahbaz — COMMENTED** [review]
Good work so far.
You implement the requirements effectively but some of them are still missing.

**Review by Javed-Shahbaz — CHANGES_REQUESTED** [review]
Please follow DRY, KISS, and YAGNI, to minimize the code in style.css file.

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
+* {
```
Comment (Javed-Shahbaz) [inline src/style.css:1]: Please follow the Dry, KISS AND YAGNI rule by minimizing your repeated code.

**Review by Javed-Shahbaz — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡

Hi @Jean-luois:

Great work so far 👏 👏
You've done well implementing some of the requirements, but there are still some which aren't addressed yet. 🥲 🥲

[image](https://media.tenor.com/Pxh2kc7GmDoAAAAM/dog-so-close.gif)


Highlights
No linter errors. ✔️
GitHub flow is followed. ✔️
PR has a good title and summary. ✔️
Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/style.css (hunk 1 above)
Comment (Mukaba) [reply src/style.css:1]: thanks @javed

**Review by Javed-Shahbaz — APPROVED** [review]
Status: APPROVED 🔥 🔥
image

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**